pyqir-generator/src/python.rs

```diff
@@ -30,139 +30,125 @@ impl PyQIR {
         }
     }
 
-    fn add_measurement(&mut self, qubit: String, target: String) -> PyResult<()> {
+    #[allow(clippy::needless_pass_by_value)]
```

Comment:
It may be helpful to have a brief comment explaining these two attributes and why they're useful here.